aries-site/src/pages/foundation/date-and-time.mdx

```diff
@@ -0,0 +1,272 @@
+
+For log files, there are two options:
+
+1. Always output timestamps in UTC, indicating UTC in the output.
+1. Allow the user to specify a default timezone and always output timestamps in that timezone, specifying the timezone in the output.
+
+### Locale formatting
+
+Applying the user experience principles of "personalization" and "don't make me think," in most cases dates and times should be presented based on the user's locale as specified in their browser and system preferences.
+
+Date and time are formatted differently depending on the language and region. Some aspects that are locale-dependent include date and time ordering, separators, and 12-hour vs 24-hour clocks.
+
+There are various [Javascript Date methods](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat) that can be leveraged to convert from UTC to the user's locale.
+
+Whenever possible, present date and time according to the user's locale.
+
+export const localeDate = new Date('2023-09-15T12:00:00.000Z');
+
+export const localeFormat = {
+  dateStyle: 'full',
+};
+
+| Locale | Date                                                            | Code snippet (where date is a Javascript Date object)                           |
+| ------ | --------------------------------------------------------------- | ------------------------------------------------------------------------------- |
+| en-US  | {Intl.DateTimeFormat('en-US', localeFormat).format(localeDate)} | <Text>`Intl.DateTimeFormat('en-US', { dateStyle: 'full' }).format(date)`</Text> |
+| en-AU  | {Intl.DateTimeFormat('en-AU', localeFormat).format(localeDate)} | <Text>`Intl.DateTimeFormat('en-AU', { dateStyle: 'full' }).format(date)`</Text> |
+| de     | {Intl.DateTimeFormat('de', localeFormat).format(localeDate)}    | <Text>`Intl.DateTimeFormat('de', { dateStyle: 'full' }).format(date)`</Text>    |
+
+### Timezones
+
+In addition to formatting a date according to the user's locale, present time in the user's timezone and display the timezone alongside the time stamp for clarity.
+
+In instances when a user's timezone cannot be determined, output the date and time in UTC+0.
+
+<Notification
+  status="info"
+  message="The same moment in time will be presented differently based on the user's timezone and locale."
+  width={{ max: 'large' }}
+  margin={{ bottom: 'medium' }}
+/>
+
+<BestPracticeGroup>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'do',
+      message:
+        'Utilize the browser locale to format and update the date and time for users, providing a personalized experience.',
+    }}
+  >
+    <TimezoneExample />
+  </Example>
+  <Example
+    height="none"
+    bestPractice={{
+      type: 'dont',
+      message: `Don't present a date in a different timezone than that of the user. This leads to confusion, increased cognitive load, and sets users up for failure.`,
```

Comment:
Originally, I had the same visual examples for the do and don't sections. Received some feedback saying that it was a bit confusing that both examples had the same time for the LA times, leading me to change the don't section into a completely different starting time for LA. I'm indifferent either way but wanted to bring this up.

Reply:
Gotcha, I can revisit.

Reply:
Done.